rotkehlchen/data_import/utils.py

```diff
@@ -13,26 +15,66 @@
 from rotkehlchen.exchanges.data_structures import AssetMovement, MarginPosition, Trade
 from rotkehlchen.history.events.structures.base import HistoryBaseEntry
 from rotkehlchen.history.events.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.logging import RotkehlchenLogsAdapter
 from rotkehlchen.serialization.deserialize import deserialize_asset_amount, deserialize_timestamp
 from rotkehlchen.types import AssetAmount, Fee, Location, TimestampMS
 
 ITEMS_PER_DB_WRITE = 400
+MAX_ERROR_PERCENT = 0.2  # max percent of messages to total entries
+MIN_ENTRIES = 50  # mininmum number of entries before checking MAX_ERROR_PERCENT
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
 
 
 class BaseExchangeImporter(ABC):
-    def __init__(self, db: DBHandler) -> None:
+    def __init__(self, db: DBHandler, name: str = '') -> None:
```

Comment:
Why give it an empty string default value. Shouldn't it always be given? Please make it a positional argument.